src/java.base/share/classes/java/lang/reflect/Constructor.java

```diff
@@ -605,9 +605,16 @@ String getSignature() {
     }
 
     @Override
-    boolean handleParameterNumberMismatch(int resultLength, int numParameters) {
+    boolean handleParameterNumberMismatch(int resultLength, Class<?>[] parameterTypes) {
+        int numParameters = parameterTypes.length;
         Class<?> declaringClass = getDeclaringClass();
-        if (declaringClass.isEnum() ||
+        if (
+            declaringClass.isEnum()
```

Comment:
Please format as
if (declaringClass.isEnum()) {
...

Reply:
Changed the formatting.